draft-ietf-httpbis-resumable-upload.md

```diff
@@ -359,17 +352,19 @@ upload-offset: 200
 
 The client MAY automatically attempt upload resumption when the connection is terminated unexpectedly, or if a server error status code between 500 and 599 (inclusive) is received. The client SHOULD NOT automatically retry if a client error status code between 400 and 499 (inclusive) is received.
 
-# Upload Cancellation Procedure {#upload-cancellation}
+# Upload Cancellation {#upload-cancellation}
+
+If the client wants to terminate the transfer without the ability to resume, it can send a `DELETE` request to the upload resource. Doing so is an indication that the client is no longer interested in continuing the upload, and that the server can release any resources associated with it.
 
-If the client wants to terminate the transfer without the ability to resume, it MAY send a `DELETE` request to the upload URL, as obtained from the Upload Creation Procedure ({{upload-creation}}). It is an indication that the client is no longer interested in uploading this body and the server can release resources associated with this upload URL. The client MUST NOT initiate this procedure without the knowledge of server support.
+The client MUST NOT initiate cancellation without the knowledge of server support.
```

Comment:
If we are removing the sentence above (knowledge of server support), we can remove this one as well

Reply:
```suggestion
```